src/transaction.rs

```diff
@@ -76,17 +78,7 @@ impl Default for Transaction {
         // The Return op is mandatory for the execution of any context
         let script = Opcode::RET(0x10).to_bytes().to_vec();
 
-        Transaction::script(
-            0,
-            MAX_GAS_PER_TX,
-            0,
-            0,
-            script,
-            vec![],
-            vec![],
-            vec![],
-            vec![],
-        )
+        Transaction::script(0, 1, 0, 0, script, vec![], vec![], vec![], vec![])
```

Comment:
just to double check, `gas_limit` was previously `100_000_000` now it's `1`, that's fine since it's just a default value?

Reply:
This is a dummy default tx with 0 gas price and byte price, so we don't really need any gas limit here.